app/models/journeys/further_education_payments/session_answers.rb

```diff
@@ -96,6 +96,17 @@ def fe_provider_closed?
 
         school.closed?
       end
+
+      def award_amount
+        case teaching_hours_per_week
+        when "more_than_12"
+          school.eligible_fe_provider.max_award_amount
+        when "between_2_5_and_12"
+          school.eligible_fe_provider.max_award_amount / 2
```

Comment:
Do we need to round this to either a whole number or 2 decimals? I realise the view helper will do this for display purposes, but we might want to implement it here instead for safety.

Reply:
i think if we just took the CSV uploaded values instead of dividing by 2 we could have avoided this scenario in the first place. I feel now we've simply created more work and complexity for ourselves. my recommendation would be to take the static values that are fed to us rather than having to write any code. otherwise i now have heaps of questions:
- what are the rounding rules?
- do we always display pence even when zero pence?
- are the rounding rules merely for display purposes? or also used for award amount?

Reply:
As discussed, we will supply the lower amount in the CSV file.